webpack.config.js

```diff
@@ -83,12 +83,12 @@ module.exports = {
       "./node_modules/prismjs/themes/prism-solarizedlight.css",
       "./node_modules/font-awesome/css/font-awesome.min.css",
     ],
-    "clarity-ui/clarity-ui": "./src/clarity-angular/main.scss",
-    "clarity-ui/clarity-ui.min": "./src/clarity-angular/main.scss",
-    // "clarity-ui/clarity-ui-dark": "./src/clarity-angular/dark-theme.scss",// Uncomment for dark dev
-    // "clarity-ui/clarity-ui-dark.min": "./src/clarity-angular/dark-theme.scss", // Uncomment for dark dev
-    "clarity-icons/clarity-icons": "./src/clarity-icons/clarity-icons.scss",
-    "clarity-icons/clarity-icons.min": "./src/clarity-icons/clarity-icons.scss"
+    "clr-ui/clr-ui": "./src/clr-angular/main.scss",
+    "clr-ui/clr-ui.min": "./src/clr-angular/main.scss",
+    // "clr-ui/clr-ui-dark": "./src/clr-angular/dark-theme.scss",// Uncomment for dark dev
```

Comment:
This should stay commented out?

Reply:
I'm leaving these in as a convenience so that when / if others have to work on dark theme they can uncomment and run `npm start` without having to remember how to modify. 

Maybe its worth adding a webpack config for developing on the dark theme and adding two start commands `npm start:light` && `npm start:dark` ? 

If you think it's worth adding, I'll rather add it in a separate PR though.